api/nodes/serializers.py

```diff
@@ -47,6 +47,30 @@ class NodeLicenseSerializer(RestrictedDictSerializer):
     year = ser.CharField(allow_blank=True, read_only=True)
 
 
+class NodeCitationSerializer(ser.Serializer):
+    id = IDField(source='_id', read_only=True)
+    title = ser.CharField(allow_blank=True, read_only=True)
+    author = ser.ListField(read_only=True)
+    publisher = ser.CharField(allow_blank=True, read_only=True)
+    type = ser.CharField(allow_blank=True, read_only=True)
+    doi = ser.CharField(allow_blank=True, read_only=True)
+
+    links = LinksField({'self': 'get_absolute_url'})
+
+    def get_absolute_url(self, obj):
+        return obj['URL']
```

Comment:
I was going to ask if this works, because we usually use lower-case keys, but it does:
```
{'URL': 'localhost:5000/26ynf',
 'author': [{'family': u'Frazier', 'given': u'Matt'}],
 'id': u'26ynf',
 'issued': {'date-parts': [[2015, 11, 16]]},
 'publisher': 'Open Science Framework',
 'title': u'OSFO4',
 'type': 'webpage'}
```